.travis-deploy.sh

```diff
@@ -17,5 +17,19 @@ if [[ "${TRAVIS_PULL_REQUEST}" = "false" && "$TRAVIS_OS_NAME" = "linux" && $TASK
     rsync -avz --delete --exclude .git/ ../packagename/ ./
     git add -A
     git commit -m "Update rendered version to ""$TRAVIS_COMMIT"
+
+    # Add astropy_helpers manually at the version in the cookiecutter template
+    git submodule add https://github.com/astropy/astropy-helpers astropy_helpers || true
```

Comment:
why would you need ``|| true``?

Reply:
because once this is in the rendered branch the submodule will come with the clone (right?!)

Reply:
I'm not sure I follow. If astropy-helpers is already in the rendered then you don't need to init it, and you already have ah_bootstap, too, don't you? probably I have to run this script locally to see the full logic.

Reply:
oh, ok. Then just lets agree that the script needs preferably to be cleaned up after the first build, or just put all of those properly in an if statement.

Reply:
You are more than welcome to improve my bash, you have many more skills with it than I do.